hack/test-cmd.sh

```diff
@@ -257,6 +251,12 @@ echo "config files: ok"
 export KUBECONFIG="${HOME}/.kube/non-default-config"
 export CLUSTER_ADMIN_CONTEXT=$(oc config view --flatten -o template -t '{{index . "current-context"}}')
 
+pushd "$OS_ROOT" > /dev/null
+restore_nullglob=$(shopt -p nullglob)
```

Comment:
`shopt -p nullglob` prints the command that will set `nullglob` to its current setting (typically, that command is `shopt -u nullglob`, i.e., disabled).  We're setting `nullglob` so that `test/cmd/**/*.sh` expands to nothing if the glob matches no files, rather than assigning the literal string `test/cmd/**/*.sh` to `tests` (which is the weird default behaviour), and then restoring `nullglob` on the off chance that some code later on relies on `nullglob`'s being set as it was before we fiddle with it here.